### PR TITLE
Adding url model

### PR DESCRIPTION
**Pull request by laudickson**

### CODE REVIEW

**Comment by laudickson** [comment]
don't worry about `lib/external_api_request.rb`

File: lib/external_api_request.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
+require 'active_support/core_ext/hash/compact'
```
Comment (laudickson) [inline lib/external_api_request.rb:1]: don't worry about this file

**Comment by laudickson** [comment]
will also need to write tests, that'll be part of this PR

**Comment by K9Advantix53** [comment]
this PR by any other name would still be as dank

**Comment by jscho13** [comment]
Super gucci bro

**Pull request merged**